### PR TITLE
build: reduce jelbourn presence in code owners

### PR DESCRIPTION
**Pull request by jelbourn**
Updating this to shuffle more code reviews from myself to @andrewseguin

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -124,12 +124,12 @@
 /src/material-experimental/mdc-table/**            @andrewseguin
 /src/material-experimental/mdc-theming/**          @mmalerba
 /src/material-experimental/mdc-typography/**       @mmalerba
-/src/material-experimental/menubar/**              @jelbourn @andy9775
+/src/material-experimental/menubar/**              @andrewseguin @andy9775
```
Comment (josephperrott) [inline .github/CODEOWNERS:127]:
```suggestion
/src/material-experimental/menubar/**              @andrewseguin
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -147,39 +147,39 @@
...
+/src/dev-app/bottom-sheet/**                       @andrewseguin @crisbeto
+/src/dev-app/button-toggle/**                      @andrewseguin
 /src/dev-app/mdc-autocomplete/**                   @crisbeto
-/src/dev-app/button/**                             @jelbourn
-/src/dev-app/card/**                               @jelbourn
+/src/dev-app/button/**                             @andrewseguin
+/src/dev-app/card/**                               @andrewseguin
 /src/dev-app/cdk-experimental-combobox/**          @jelbourn @nielsr98
 /src/dev-app/cdk-experimental-listbox/**           @jelbourn @nielsr98
 /src/dev-app/cdk-experimental-menu/**              @jelbourn @andy9775
```
Comment (josephperrott) [inline .github/CODEOWNERS:159]:
```suggestion
/src/dev-app/cdk-experimental-menu/**              @jelbourn
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -207,47 +207,47 @@
 /src/dev-app/menubar/**                            @jelbourn @andy9775
```
Comment (josephperrott) [inline .github/CODEOWNERS:207]:
```suggestion
/src/dev-app/menubar/**                            @jelbourn
```

**Review by josephperrott — APPROVED** [review]
LGTM

**Review by andrewseguin — APPROVED** [review]

**Comment by jelbourn** [comment]
Removed Andy and Niels

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,86 +1,86 @@
...
+/src/material/core/selection/pseudo*/**            @crisbeto @andrewseguin
+/src/material/core/style/**                        @andrewseguin
+/src/material/core/theming/**                      @andrewseguin @jelbourn
 /src/material/core/typography/**                   @crisbeto
-/src/material/core/util/**                         @jelbourn
+/src/material/core/util/**                         @andrewseguin
 
 # Miscellaneous components
 /src/google-maps/**                                @mbehrlich
 /src/youtube-player/**                             @nathantate
```
Comment (crisbeto) [inline .github/CODEOWNERS:66]: Should we have more people on `google-maps` and `youtube-player`?

Comment (jelbourn) [reply .github/CODEOWNERS]: Added you

**Pull request merged**